README.md

```diff
@@ -7,36 +7,47 @@ To allow exchange of cross-linguistic data and decouple development of tools and
+The main types of cross-linguistic data we are concerned with here are wordlists and structure datasets which are used in historical linguistics and linguistic typology.
 
 
 ## Design goals
 
 - Data should be both editable "by hand" and amenable to reading and writing by software.
-- UTF-8 encoded text files.
-- Reference entities rather than duplicate.
-- IDs should be resolvable HTTP URLs if possible.
-- Compatibility with existing tools, standards and practice should alsways be kept in mind.
+- UTF-8 encoded line-based (ideally LF) text files.
+- Reference entities rather than duplicate (i.e. use `long form tables`, sometimes called [`narrow form`][http://en.wikipedia.org/wiki/Wide_and_narrow_data])
+- IDs should be resolvable HTTP URLs if possible. If not, they should be documented in the metadata
+- Compatibility with existing tools, standards and practice should always be kept in mind.
 
 
 ## Core format specification
 
 A cross-linguistic dataset is encoded in the following set of files:
 
-- The core data file, encoded in csv, 
-- additional metadata provided as JSON file following the guidelines of the [Model for Tabular Data and Metadata on the Web](http://www.w3.org/TR/tabular-data-model/#standard-file-metadata), 
-- sources - if not referenced by Glottolog ID - supplied as BibTeX file (with the citation keys serving as local Source IDs).
-- Examples - if not referenced - may be supplied as [*cldf* IGT](igt.md) file.
+- The core data file, encoded in csv (discussion point: should we allow tsv, be it only because it is used so much in current practice? tsv-assumptions: column separator is `\t`, quotes are always taken as literals)
+- Metadata provided as JSON file (discussion point: also allow YAML, because it is much easier to edit by hand without introducting errors) following the guidelines of the [Model for Tabular Data and Metadata on the Web](http://www.w3.org/TR/tabular-data-model/#standard-file-metadata)
+- The core data file can be amended by subsidiary `linked` files to explicate local IDs, e.g.
+	- Sources - if not referenced by Glottolog ID, or other standard bibliographic identifiers like OCLC or ISBN - supplied as BibTeX file (with the citation keys serving as local Source IDs).
+	- Languoids - if not referenced by Glottolog ID or ISO 639/3, or BCP47, supplying information on the languages references in the core data file
+	- Explications about definitions of Features and Values - if not references by external ID - can be supplied in separate linked data files
+	- Examples - if not referenced - may be supplied as [*cldf* IGT](igt.md) file.
 
 If the name of the dataset is `clds`, the respective filenames are
-- `clds.csv`
-- `clds.csv-metadata.json`
-- `clds.bib`
-- `clds.igt.csv`
 
+- `clds.csv` (or `clds.tsv`)
+- `clds-metadata.json` (or `clds-metadata.yaml`)
+
+Optionally also e.g.
+
+- `clds-references.bib`
+- `clds-languoids.csv`
```

Comment:
I guess this is intended for language metadata, for which I had a [separate proposal](https://github.com/glottobank/cldf/blob/master/language_metadata.md) you may have not seen?